src/scenes/Organizations.re

```diff
@@ -7,74 +8,188 @@ module Styles = {
       flex(1.),
       backgroundColor(String(AppConfig.theme.backgroundColor)),
     ]);
-  let content =
-    style([alignItems(FlexEnd), flexDirection(Column), width(Pct(90.))]);
 
-  let contentTitle = style([fontSize(Float(16.))]);
+  let buttonWrapper = style([marginTop(Pct(10.))]);
+
+  let content = style([width(Pct(80.))]);
+
+  let contentTitle = style([fontWeight(`_700), fontSize(Float(16.))]);
 
   let contentDetail = style([fontSize(Float(14.))]);
 
-  let buttonWrapper = style([marginTop(Pct(10.))]);
+  let contentDate = style([fontSize(Float(12.))]);
 };
 
+module Query = [%graphql
+  {|
+    query AppQuery {
+      currentUser {
+        id
+        email
+        role
+        summary {
+          unread
+          total
+          projects
+          organizations
+        }
+        notifications {
```

Comment:
shouldn't be a query against organizations?

Reply:
yeah, but we don't have it yet

Reply:
answered on discord